src/main/drivers/vtx_rtc6705.c

```diff
@@ -31,6 +31,7 @@
 
 #include "common/maths.h"
 
+#include "io/vtx_rtc6705.h"
```

Comment:
Drivers should not be 'calling up'. If these defines are needed in the driver layer, then they will have to be defined there, and the io layer will have to include the driver header to access them.

Reply:
I concur with @mikeller